scripts/scil_connectivity_math.py

```diff
@@ -41,29 +42,33 @@ def _build_arg_parser():
                    choices=OPERATIONS.keys(),
                    help='The type of operation to be performed on the '
                         'matrices.')
-    p.add_argument('inputs', nargs='+',
+    p.add_argument('in_matrices', nargs='+',
                    help='The list of matrices files or parameters.')
     p.add_argument('out_matrix',
                    help='Output matrix path.')
 
     p.add_argument('--data_type',
                    help='Data type of the output image. Use the format: '
                         'uint8, float16, int32.')
+    p.add_argument('--exclude_background', action='store_true',
```

Comment:
Sneaking in a new feature in a refactoring PR ;)

Reply:
It already in the master, don't know why its green (But yeah I do that often)

Reply:
Fun fact: it's **not** in the master branch, you must have started your branch from another that isn't merged yet... Be careful, should this really be merged in this PR?

Reply:
I just checked, it's there...
https://github.com/scilus/scilpy/blob/master/scripts/scil_image_math.py

Reply:
Your "new feature" is in `scil_connectivity_math.py`

Reply:
It works as intended, but obviously less intuitive as the image background is a quite clearer concept.
For matrices it simply mean that everywhere you had zeros before you will still have 0 after, even if you do `scil_connectivity_math.py addition 17 *.npy masked_addition.npy --exclude background`
This would add 17 to all nonzeros values

Which would be equivalent to finding all zeros (like doing a binarization of all your npy, do an intersection of all binary mask, do the command `scil_connectivity_math.py addition 17 *.npy addition.npy` followed by `scil_connectivity_math.py multiplication addition.npy mask.npy masked_addition`

This is maybe overly specific as a use case, but I need it some time so I say we need it :100: